### ISSUE TITLE
Write "Use my location" as a button

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#21247]
## Issue Description
As a keyboard user, I want to be able to use the space bar to activate "Use my location" (rather than just the enter key) so that it is consistent with the usual keyboard navigation. 
As a Veteran using voice assistive technology, I want to be able to ask my tech to "Click on the use my location button" instead of "Click on the use my location link." so that I can access it better.  

"Use my location" should be written as a button <button> instead of a link <a> since the user is not navigating to a new page

---
## Tasks
- [ ] Write "Use my location" as a button. 
- [ ] Fix visual behavior of clear input and use my location button

## Acceptance Criteria
- [ ] Veterans navigating the Facility Locator using a keyboard can activate "Use my location" with the space bar. 
- [ ] Veterans using voice assistive technology ask the tech to "Click on the Use my location button" instead of "Click on the Use my location link."
- [ ] Clear input and use my location buttons do not shift to the right

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by tlei123** [comment]
hi @lildziner , would you expect to have updated designs for this button?

if yes, will you be updating these two places in Zeplin?
- [Mobile](https://app.zeplin.io/project/600877fdb269d8a4f1f44efd/screen/600879364afaa6a6a2a8a827)
- [Desktop](https://app.zeplin.io/project/600877fdb269d8a4f1f44efd/screen/6008792b280d84a85bc6c015)

**Comment by mmiddaugh** [comment]
We're not updating the design @tlei123 - just the coding

**Comment by mmiddaugh** [comment]
Issue moved to [department-of-veterans-affairs/va.gov-vfs-teams #55](https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/issues/55) via [**ZenHub**](https://www.zenhub.com/)
